### ISSUE TITLE
[summary] REPORT vs EPOCH

### ISSUE THREAD

**Issue by mmminji** [mmminji/RTV-project#9]
- 1epoch는 len(dataloader)만큼 학습 = iter
- REPORT 수만큼 iter 돌면 loss, time 기록 + eval
   (loss 앞에 뜨는 수 * REPORT) 만큼 iter가 돌았다는 의미
- **즉, REPORT는 몇 iter마다 eval 할 것인지 결정**

### 1 EPOCH에  1 REPORT : (train data size / batch size)로 REPORT 설정
train data size는 `original_data/train.summary` 길이
